packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -176,8 +226,11 @@ exports.sourceNodes = async (
     node.internal.contentDigest = createContentDigest(node)
     createNode(node)
   }
+
+  return reporter.info(`Processed Drupal data`)
 }
 
+// This is maintained for legacy reasons and will eventually be removed.
```

Comment:
Shall we add a deprecation log inside the call saying that it will eventually be removed with a link to a migration path?